packages/experience-builder-sdk/src/blocks/editor/VisualEditorBlock.tsx

```diff
@@ -130,6 +130,18 @@ export const VisualEditorBlock = ({
             ...acc,
             [variableName]: transformContentValue(value, variableDefinition),
           };
+        } else if (variableMapping.type === 'ComponentValue') {
```

Comment:
We might not need this check `if (variableMapping.type === 'ComponentValue')` if we return the default value if there's nothing in the `componentInstanceUnboundValues` object [here](https://github.com/contentful/experience-builder/blob/76713e64584d10f7fe6ef7eef7c3cbeb951dee53/packages/experience-builder-sdk/src/core/editor/designComponentUtils.ts#L45).

Because what's happening in this check is almost the same as what's happening on line 146. If when we deserialize the nodes, we insert a value or default value in the node unbound value object, we probably won't need the extra check here.